cayenne-server/src/main/resources/org/apache/cayenne/schema/11/dbimport.xsd

```diff
@@ -73,6 +71,8 @@
 
     <xs:complexType name="container" abstract="true">
         <xs:sequence>
+            <xs:element name="catalog" minOccurs="0" maxOccurs="unbounded" type="dbi:catalog"/>
```

Comment:
This change would allow to place `schema` and `catalog` tags inside any type that inherits from `container`. And that includes `schema` that shouldn't allow that.